thorlcr/thorutil/thbuf.cpp

```diff
@@ -1608,7 +1609,11 @@ class CSharedWriteAheadMem : public CSharedWriteAheadBase
     }
     virtual size32_t rowSize(const void *row)
     {
-        return meta->getRecordSize(row); // space in mem.
+        if (!row)
```

Comment:
@jakesmith The change all looks good, happy to merge as is.
Was the idea behind moving the serializer to also move this rowSize() function into the base class?  (If so it come non-virtual.)

Reply:
the disk vs mem implementations are subtly different, so can't common up into to base and make non-virtual.